Header header

int32 	class
float32 confidence
int32 	centerX
int32 	centerY
int32 	boxHeight
int32 	boxWidth
int32 	imgHeight
int32 	imgWidth

